indico/modules/users/__init__.py

```diff
@@ -41,6 +41,7 @@
     'timezone': None,
     'force_timezone': False,  # always use the user's timezone instead of an event's timezone
     'show_past_events': False,
+    'name_ordering': NameOrdering.first_last,
```

Comment:
Right now retrieving this setting will return just a plain integer instead of the actual enum value. This often works fine but we now support properly handling enums in a SettingsProxy using the [EnumConverter](https://github.com/indico/indico/blob/devel/indico/core/settings/converters.py#L77). Just search for it and you'll find an example how it's used.